libraries/LittleFS/examples/SpeedTest/SpeedTest.ino

```diff
@@ -14,6 +14,24 @@
 // How large of a file to test
 #define TESTSIZEKB 512
 
+// Format speed in bytes/second.  Static buffer so not re-entrant safe
+const char *rate(long start, long stop, long bytes) {
+  static char buff[64];
+  if (stop == start) {
+    strcpy_P(buff, PSTR("Inf b/s"));
+  } else {
+    float r = 1000.0 * (float)bytes / (float)(stop - start);
```

Comment:
`if (start > stop)`? (and to make reader note of the signed type)

Reply:
Fair points.   In the test as is there’s no chance of these things happening, but it’s a good idea to provide good examples.  So, this should use uint32s as parameters since that’s what millis returns.

It’s perfectly fine for start to be larger than end.  When you start at, say 2^32-1 millis you’ll end at something lower.  The unsigned difference will still be the correct runtime.

I’m on the phone right now, but will do the slight modification suggested in a PR later today.